src/Opg/Lpa/DataModel/AbstractData.php

```diff
@@ -247,7 +246,7 @@ public function validate(array $properties = [], array $groups = [])
     public function toArray($dateFormat = 'string')
     {
         // @codeCoverageIgnoreStart
-        if ($dateFormat == 'mongo' && !class_exists('\MongoDate')) {
+        if ($dateFormat == 'mongo' && !class_exists(MongoDate::class)) {
```

Comment:
I don't think you can use MongoDate::class here as it will cause an error if not present. Perhaps we should just remove the class_exists check?